### DISCUSSION TITLE
Nickname in nameplate

### DISCUSSION THREAD

**Discussion by HeartlessAUS** [Zrips/CMI#1877] [Feature Requests]
Can we get a way to set nicknames in the players nameplates? thanks!

---

**Comment by mrfloris** [comment]
Yes pls, this would be nice.
ie
`/cmi nameplace -c:&6 -pref:* -suff:' &3king' -n:[playerNickName]`

**Comment by HeartlessAUS** [comment]
or just based on /nick as an option in the config

**Comment by mrfloris** [comment]
As long as I can control the player's details through console with commands on a per player basis
